LiteDB/Engine/EngineSettings.cs

```diff
@@ -66,6 +66,11 @@ public class EngineSettings
         /// </summary>
         public bool Upgrade { get; set; } = false;
 
+        /// <summary>
+        /// Is used to transform a bsonvalue from the database on read. This can be used to upgrade data from older versions.
```

Comment:
```suggestion
         /// Is used to transform a <see cref="BsonValue"/> from the database on read. This can be used to upgrade data from older versions.
```